bindings/pyfiction/include/pyfiction/algorithms/simulation/sidb/determine_physically_valid_parameters.hpp

```diff
@@ -0,0 +1,63 @@
+//
+// Created by Jan Drewniok on 07.05.24.
+//
+
+#ifndef PYFICTION_DETERMINE_PHYSICALLY_VALID_PARAMETERS_HPP
+#define PYFICTION_DETERMINE_PHYSICALLY_VALID_PARAMETERS_HPP
+
+#include "pyfiction/documentation.hpp"
+#include "pyfiction/types.hpp"
+
+#include <fiction/algorithms/simulation/sidb/determine_physically_valid_parameters.hpp>
+#include <fiction/algorithms/simulation/sidb/operational_domain.hpp>
+
+#include <pybind11/pybind11.h>
+#include <pybind11/stl.h>
+
+namespace pyfiction
+{
+
+namespace detail
+{
+
+template <typename Lyt>
+void determine_physically_valid_parameters(pybind11::module& m)
+{
+    using namespace pybind11::literals;
+
+    m.def("determine_physically_valid_parameters", &fiction::determine_physically_valid_parameters<Lyt>, "cds"_a,
+          "params"_a = fiction::operational_domain_params{});
+
+    m.def("find_parameter_point_with_tolerance",
```

Comment:
@marcelwa Do you know if this can work? The corresponding unit test does not work.

Reply:
I would need a bit more context here. Which function are you talking about? What is the error message you're getting?